start-ocp-console.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+mkdir -p ocp-console
+oc process -f ocp-console-oauth-client.yaml | oc apply -f -
+
+oc get oauthclient console-oauth-client -o jsonpath='{.secret}' > ocp-console/console-client-secret
+
+oc get secrets -n default --field-selector type=kubernetes.io/service-account-token -o json | \
+    jq '.items[0].data."ca.crt"' -r | python -m base64 -d > ocp-console/ca.crt
+
+
+CONSOLE_IMAGE=${CONSOLE_IMAGE:="quay.io/openshift/origin-console:latest"}
```

Comment:
I wonder if we should pin this at the 4.10 level by default, or maybe parameterize it?

Reply:
Sounds good, by default it will download 4.10, but we can still use custom version by defining `CONSOLE_VERSION`